apps/server/internal/runexec/docker_integration_test.go

```
@@ -0,0 +1,371 @@
+	if err != nil {
+		t.Fatal(err)
+	}
+	dockerRuntime, err := dockerruntime.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	services, err := app.NewServicesWithRuntimes(database, materializer, map[string]runtimepkg.Implementation{"docker": dockerRuntime})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer services.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `services.Close` is not checked

(errcheck)